libraries/debug_eos_vm/include/debug_eos_vm/dwarf.hpp

```diff
@@ -2,6 +2,7 @@
 
 #include <eosio/stream.hpp>
 #include <memory>
+#include <optional>
```

Comment:
@tbfleming without this line clang was failing. is it fine to add this change?

Reply:
yes